miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultContainerService.java

```diff
@@ -81,6 +81,11 @@ public AuthorizationManager getAuthorizationManager() {
     return authorizationManager.filterUnreadable(containers);
   }
 
+  @Override
+  public Collection<Partition> listPartitionsByPoolId(long poolId) throws IOException {
+    return containerDao.listAllPartitionsByPoolId(poolId);
+  }
+
   @Override
   public SequencerPartitionContainer get(long containerId) throws IOException, AuthorizationException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'uk.ac.bbsrc.tgac.miso.service.security.AuthorizationException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)